packages/devtools_app/test/utils_test.dart

```diff
@@ -14,12 +14,12 @@ void main() {
 
       expect(printMb(10 * MB, 0), '10');
       expect(printMb(10 * MB), '10.0');
-      expect(printMb(10 * MB, 1), '10.0');
+      expect(printMb(10 * MB), '10.0');
       expect(printMb(10 * MB, 2), '10.00');
 
       expect(printMb(1000 * MB, 0), '1000');
       expect(printMb(1000 * MB), '1000.0');
-      expect(printMb(1000 * MB, 1), '1000.0');
```

Comment:
ignore the lint for this case or remove as this line is now identical to the one above